common/templates/collection-as-table.njk

```diff
@@ -30,4 +30,24 @@
       }
     }) }}
   {% endif %}
+
+  {% if resultsAsTable.cancelled.rows.length %}
+    {% set html %}
+      <div>
+        {{ govukTable({
+          rows: resultsAsTable.cancelled.rows,
+          head: resultsAsTable.cancelled.head,
+          classes: "govuk-secondary-text-colour"
```

Comment:
Is there a reason to apply this knocked back colour to the whole table?

I think the results being nested within the details is enough to show they are linked to the cancelled action.

Reply:
This is something @Andy-Millar explicitly requested.

Reply:
Then maybe it could be something that lives in the presenter. Their purpose is to transform something into the expected output for a component. 

See the example above how the output of the presenter for active moves is passed straight into the component.

Reply:
Actually, let's just leave this for now to get this in.